packages/cinterop/src/native/realm.def

```diff
@@ -1,5 +1,6 @@
 headers = realm.h realm/error_codes.h
 headerFilter = realm.h realm/error_codes.h
+compilerOpts = -DREALM_APP_SERVICES=1
```

Comment:
What does this option do?

Reply:
It's a build option to include core's default implementation of app services api. Eventually we will be able to supply our own, so that we can build the MongoDB API without core.